repos/cray.template.repos

```diff
@@ -11,3 +11,6 @@ https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifac
 
 # SDU
 https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/sdu-rpm-stable-local/release/img_oci-shasta-2.1/sle15_sp3_ncn?auth=basic                                   sdu-rpm-stable-local                              --no-gpgcheck -p 89                  cray/sdu/sle-15-sp3
+
+# CVT
+https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/dst-rpm-mirror/csm-diags-rpm-stable-local/release/csm-diags-1.4.0/sle15_sp3?auth=basic                                    csm-diags-sle-15sp3-1.4.0                  --no-gpgcheck -p 89                   cray/cvt/sle-15sp3
```

Comment:
I heard an SP4 version was coming, assuming that's coming soon please update this link before merging.

Reply:
Addressed